eo-maven-plugin/src/main/java/org/eolang/maven/DcsDepgraph.java

```diff
@@ -0,0 +1,223 @@
+        } catch (final MojoExecutionException ex) {
+            throw new IllegalStateException(
+                String.format(
+                    "Dphgraph. Creation of the dependencies file failed for the dependency %s",
+                    origin
+                ),
+                ex
+            );
+        }
+    }
+
+    /**
+     * Creates filename for transitive dependencies file.
+     *
+     * @param dependency Dependency
+     * @return Filename
+     */
+    private static String fileName(final Dependency dependency) {
+        return String.format(
+            "%s_%s_%s%s",
+            dependency.getGroupId(),
+            dependency.getArtifactId(),
+            dependency.getVersion(),
+            ".json"
+        );
+    }
+
+    /**
+     * Dependencies uploaded from json file.
+     *
+     * @since 0.28.11
+     */
+    static class DcsJson implements Dependencies {
+
+        /**
+         * File path.
+         */
+        private final Path file;
+
+        /**
+         * The main constructor.
+         *
+         * @param file File path
+         */
+        DcsJson(final Path file) {
+            this.file = file;
+        }
+
+        @Override
+        public Collection<Dependency> all() {
+            try {
+                final List<Dependency> all = new ArrayList<>(0);
+                if (Files.exists(this.file)) {
+                    Logger.debug(this, String.format("Dependencies file: %s", this.file));
+                    final JsonReader reader = Json.createReader(Files.newBufferedReader(this.file));
+                    final JsonArray artifacts = reader.readObject()
+                        .getJsonArray("artifacts");
+                    for (final JsonValue artifact : artifacts) {
+                        final JsonObject obj = artifact.asJsonObject();
+                        final String group = obj.getString("groupId");
+                        final String id = obj.getString("artifactId");
+                        final String version = obj.getString("version");
+                        final String scope = obj.getJsonArray("scopes").stream()
+                            .map(JsonValue::toString)
+                            .findFirst().orElseThrow(IllegalStateException::new);
+                        final Dependency dependency = new Dependency();
+                        dependency.setGroupId(group);
+                        dependency.setArtifactId(id);
+                        dependency.setVersion(version);
+                        dependency.setScope(scope);
+                        all.add(dependency);
+                    }
+                }
+                return all;
+            } catch (final IOException | IllegalStateException ex) {
+                throw new IllegalStateException(
+                    String.format(
+                        "Exception happens during reading the dependencies from json file %s. Probably the file is absent or you have a wrong json format",
```

Comment:
@volodya-lombrozo for the future, try to enclose `%s` inside texts with single quotes. This will help readers separate your enclosures from the rest of the text:

```
... reading from JSON file '%s`, probably the file...
```

Also, try to use ONE sentence as error message, without a dot inside (and no dot at the end)